SU2_CFD/include/driver_structure.hpp

```diff
@@ -1487,3 +1534,175 @@ class CMultiphysicsZonalDriver : public CDriver {
   void Transfer_Data(unsigned short donorZone, unsigned short targetZone);
 
 };
+
+/*!
+ * \class CSinglezoneDriver
+ * \brief Class for driving single-zone solvers.
+ * \author R. Sanchez
+ * \version 6.0.1 "Falcon"
+ */
+class CSinglezoneDriver : public CDriver {
```

Comment:
I will be doing soon a cleaning PR, so I'll keep an eye on the methods that are no longer needed. First, I thought it would be good to have the additions merged in so the community can adapt easier.

Adjoint versions will be coming later. @oleburghardt has been doing some great work on that front, but we still need to incorporate the FSI adjoint to this new infrastructure. For now, this covers only primal solvers.